src/pages/TeamSettings/Members/Members-Table.vue

```diff
@@ -268,14 +273,28 @@ export default {
 
       <!-- ROLE -->
       <template #item.role="{ item }">
-        <v-chip small dark :color="roleColorMap[item.role] || 'secondaryLight'">
-          {{ roleMap[item.role] || 'Unknown' }}
+        <v-chip
+          small
+          dark
+          :color="
+            !roleColorMap[item.role]
+              ? 'accentPink'
+              : roleColorMap[item.role] || 'secondaryLight'
+          "
+        >
+          {{ roleMap[item.role] || item.role }}
         </v-chip>
       </template>
 
       <!-- ACTIONS -->
-      <template v-if="isTenantAdmin" #item.actions="{ item }">
-        <v-tooltip v-if="hasPermission('feature', 'basic-rbac')" bottom>
+      <template v-if="permissionsCheck" #item.actions="{ item }">
+        <v-tooltip
+          v-if="
+            hasPermission('feature', 'basic-rbac') &&
```

Comment:
May need to check for feature:custom-role also.  (Or remove the basic-rbac check)

Reply:
i think `basic-rbac` is fine for now - this needs to have the basic-rbac check for sure and since this isn't creating a custom role there shouldn't be any blocker in the backend.

Reply:
Ah Ok. Does that mean that all licenses that have custom-role will also have basic-rbac?

Reply:
That's something I'll raise cause they should